GiniVision/Classes/Networking/Extensions/GiniScreenAPICoordinator+Networking.swift

```diff
@@ -37,15 +37,15 @@ extension GiniScreenAPICoordinator {
         }
     }
     
-    var apiService: APIServiceProtocol? {
+    var documentService: DocumentServiceProtocol? {
         get {
-            return objc_getAssociatedObject(self, &AssociatedKey.apiService) as? APIServiceProtocol
+            return objc_getAssociatedObject(self, &AssociatedKey.documentService) as? DocumentServiceProtocol
```

Comment:
I don't understand why we need the runtime here.

Reply:
Because it is the only way to add stored properties to protocols.

Reply:
Is it absolutely necessary to have it in an extension then?

Reply:
Yes. It is the optional Networking package, so some classes wouldn't be available in the Core package.